# DepthImageOctomapUpdater is much faster! (~ 12-50-200 ms vs 20-180-500 ms) min-avr-max
# especially when point_subsample is set to 1 !!! 

sensors:
  # - sensor_plugin: occupancy_map_monitor/DepthImageOctomapUpdater
  #   image_topic: /camera_right/depth/image_raw
  #   queue_size: 1
  #   near_clipping_plane_distance: 0.1
  #   far_clipping_plane_distance: 1.3
  #   shadow_threshold: 0.2
  #   padding_scale: 1.0
  #   padding_offset: 0.06
  #   max_update_rate: 0

  # - sensor_plugin: occupancy_map_monitor/DepthImageOctomapUpdater
  #   image_topic: /camera_left/depth/image_raw
  #   queue_size: 1
  #   near_clipping_plane_distance: 0.1
  #   far_clipping_plane_distance: 1.3
  #   shadow_threshold: 0.2
  #   padding_scale: 1.0
  #   padding_offset: 0.06
  #   max_update_rate: 0

  - sensor_plugin: occupancy_map_monitor/DepthImageOctomapUpdater
    image_topic: /camera_top/depth/image_raw
    queue_size: 1
    near_clipping_plane_distance: 0.1
    far_clipping_plane_distance: 1.0
    shadow_threshold: 0.2
    padding_scale: 1.0
    padding_offset: 0.06
    max_update_rate: 0
